packages/logging-winston/README.md

```diff
@@ -96,6 +96,14 @@ winston.info(`${req.url} endpoint hit`, {
 
 The `httpRequest` proprety must be a properly formatted [`HttpRequest`][http-request-message] message.
 
+## Correlating Logs with Traces
+
+If you use [@google-cloud/trace-agent][trace-agent] module, then this module will set the Stackdriver Logging [LogEntry][LogEntry] `trace` property based on the current trace context when available. That correlation allows you to [view log entries][trace-viewing-log-entries] inline with trace spans in the Stackdriver Trace Viewer. Example:
+
+![Logs in Trace Example](/packages/logging-winston/doc/images/winston-logs-in-trace.png)
+
+If you wish to set the LogEntry `trace` property with a custom value, then set a winston metadata property for `'logging.googleapis.com/trace'`, which is exported by this module as `LOGGING_TRACE_KEY`.
```

Comment:
Could you add an example for this?

Reply:
And likewise for Bunyan.

Reply:
Done. I used ES6 enhanded object literals to make the examples a bit simpler, hope that was OK.